cosmos-sdk-proto/src/prost/COSMOS_SDK_COMMIT

```diff
@@ -1 +1 @@
-v0.40.0-rc3
\ No newline at end of file
+590358652cc1cbc13872ea1659187e073ea38e75
```

Comment:
The main drawback of this approach, at least as implemented right now, is it's sourcing this value from `.git/modules/cosmos-sdk/HEAD` which contains a raw commit hash instead of a nicely formatted tag name.

See the updated `output_sdk_version` below:

https://github.com/cosmos/cosmos-rust/pull/12/files#diff-58406f56aa9701745b57c8af4adb067401ed9cb97ee7cad24938a905f8314f36R63-R66